Build errors on Mac

I am testing on an M4 Mac running Sequoia 15.6.1. I am trying to build the example using `go build -tags purego ./examples/triangle/`.

I get a build error when I do so:

```
# github.com/go-webgpu/goffi/ffi
../../../go/pkg/mod/github.com/go-webgpu/goffi@v0.3.1/ffi/dl_darwin.go:57:20: undefined: dl.Dlopen
../../../go/pkg/mod/github.com/go-webgpu/goffi@v0.3.1/ffi/dl_darwin.go:91:19: undefined: dl.Dlsym
../../../go/pkg/mod/github.com/go-webgpu/goffi@v0.3.1/ffi/dl_darwin.go:140:12: undefined: dl.Dlclose
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.1] - 2025-12-23
+
+### Fixed
+- **macOS Apple Silicon (ARM64) support** — Updated goffi to v0.3.2
+  - Fixes runtime failure on M1/M2/M3/M4 Macs
+  - HFA structs (NSRect, NSPoint, NSSize) now correctly passed via float registers
+  - Resolves: `darwin: failed to create NSAutoreleasePool`
+
+### Changed
+- Updated dependency: go-webgpu/goffi v0.3.1 → v0.3.2
+
 ## [0.6.0] - 2025-12-23
 
 ### Added
@@ -144,7 +155,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **Examples**
   - `examples/triangle/` — Simple triangle demo
 
-[Unreleased]: https://github.com/gogpu/gogpu/compare/v0.6.0...HEAD
+[Unreleased]: https://github.com/gogpu/gogpu/compare/v0.6.1...HEAD
+[0.6.1]: https://github.com/gogpu/gogpu/compare/v0.6.0...v0.6.1
 [0.6.0]: https://github.com/gogpu/gogpu/compare/v0.5.0...v0.6.0
 [0.5.0]: https://github.com/gogpu/gogpu/compare/v0.4.0...v0.5.0
 [0.4.0]: https://github.com/gogpu/gogpu/compare/v0.3.0...v0.4.0

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@
 
 ---
 
-## Status: v0.5.x — Full Cross-Platform Support
+## Status: v0.6.x — Full Cross-Platform Support
 
 > **All major platforms now supported!** Windows, Linux (X11 + Wayland), macOS (Cocoa + Metal).
 >
@@ -213,7 +213,7 @@ gogpu/
 
 See **[ROADMAP.md](ROADMAP.md)** for the full roadmap.
 
-**Current:** v0.5.x — Full Linux + macOS Platform Support
+**Current:** v0.6.x — Full Linux + macOS Platform Support
 
 **Recent:**
 - ✅ **Linux X11 windowing (Pure Go, ~5K LOC) — Community Testing**
@@ -232,7 +232,7 @@ See **[ROADMAP.md](ROADMAP.md)** for the full roadmap.
 
 | Project | Description | Status |
 |---------|-------------|--------|
-| [gogpu/gogpu](https://github.com/gogpu/gogpu) | Graphics framework (this repo) | **v0.5.0** |
+| [gogpu/gogpu](https://github.com/gogpu/gogpu) | Graphics framework (this repo) | **v0.6.1** |
 | [gogpu/wgpu](https://github.com/gogpu/wgpu) | Pure Go WebGPU implementation | v0.6.0 |
 | [gogpu/naga](https://github.com/gogpu/naga) | Pure Go shader compiler (WGSL → SPIR-V, MSL) | v0.5.0 |
 | [gogpu/gg](https://github.com/gogpu/gg) | 2D graphics library | v0.9.2 |

--- a/ROADMAP.md
+++ b/ROADMAP.md
@@ -16,11 +16,11 @@ Our goal is to become the **reference graphics ecosystem** for Go — comparable
 
 ---
 
-## Current State (v0.5.0)
+## Current State (v0.6.1)
 
 | Component | Version | Description |
 |-----------|---------|-------------|
-| **gogpu/gogpu** | v0.5.0 | GPU abstraction, windowing, dual backend |
+| **gogpu/gogpu** | v0.6.1 | GPU abstraction, windowing, dual backend |
 | **gogpu/wgpu** | v0.6.0 | Pure Go WebGPU (Vulkan, Metal, GLES, Software) |
 | **gogpu/naga** | v0.5.0 | WGSL shader compiler (SPIR-V, MSL) |
 | **gogpu/gg** | v0.9.2 | 2D graphics library |

--- a/go.mod
+++ b/go.mod
@@ -8,4 +8,4 @@ require (
 	golang.org/x/sys v0.39.0
 )
 
-require github.com/go-webgpu/goffi v0.3.1
+require github.com/go-webgpu/goffi v0.3.2

